lib/test.js

```diff
@@ -44,14 +50,27 @@ const test = (suite, buildConfig = config.defaultBuildConfig, options) => {
     util.run('ninja', ['-C', config.outputDir, "fix_brave_test_install_name_adblock"], config.defaultOptions)
 
     let testBinary;
+    let installerTestBinary;
     if (process.platform === 'win32') {
       testBinary = `${suite}.exe`
+      installerTestBinary = 'brave_installer_unittests.exe'
     } else {
       testBinary = suite
+      installerTestBinary = 'brave_installer_unittests'
     }
 
     // Run the tests
     util.run(path.join(config.outputDir, testBinary), braveArgs, config.defaultOptions)
+
+    if (run_brave_installer_unitests) {
+      // Replace output file arguments
+      if (options.output) {
+        braveArgs.splice(braveArgs.indexOf('--gtest_output=xml:' + options.output, 1))
+        braveArgs.push('--gtest_output=xml:brave_installer_unittests.xml')
+      }
+
+      util.run(path.join(config.outputDir, installerTestBinary), braveArgs, config.defaultOptions)
```

Comment:
just want to verify - does CI correctly handle/report the regular unit tests fail and therefore this doesn't even run and the output file is not created?

Reply:
also what does the output look like for a regular test run for a dev?

Reply:
maybe we should force a failure and not continue to run the installer tests?

Reply:
basically I'm worried that will pull an npm and hide the errors unless you scroll up and look for them

Reply:
@bridiver When brave_unit_tests has failure, it doesn't continue to run installer test.

Reply:
ok, and that's ok for CI that the other file doesn't get generated?

Reply:
Each tests generates different output files - `brave_unit_tests.xml` and `brave_installer_unittests.xml`

Reply:
but that's my point, will CI have an issue if `brave_installer_unittests.xml` is missing?

Reply:
Hmm, I think before checking `xunit([GoogleTest(pattern: "src/brave_installer_unittests.xml", deleteOutputFiles: false, failIfNotNew: true, skipNoTestFiles: false, stopProcessingIfError: false)])`, CI should check the existence of `brave_installer_unittests.xml`. is it proper way?

Reply:
Probably, we want CI to correctly report the error and not fail in some other way that might be hard to decipher

Reply:
I'm not sure what will happen with `xunit([GoogleTest(pattern: "src/brave_installer_unittests.xml", deleteOutputFiles: false, failIfNotNew: true, skipNoTestFiles: false, stopProcessingIfError: false)])` if `brave_installer_unittests.xml` doesn't exist, but I'm guessing it will end up with some CI java stack trace instead of the error we want to report for test failures

Reply:
I don't know much about `Jenkins`. maybe needs @mihaiplesa's help about how to handle this.